miso-web/src/main/webapp/scripts/instituteDefaults_ajax.js

```diff
@@ -415,23 +415,20 @@ var Institute = Institute || {
       data.sort(function (a, b) { //////////////// TODO: confirm alias??
         return (a.alias > b.alias) ? 1 : ((b.alias > a.alias) ? -1 : 0);
       });
-    } // else collection is empty, so render only the "Add New" button
+    } // else collection is empty, so render only teh "Add New" button
```

Comment:
Must've been my fault, not @djcooke . I'll be reworking Institute and Lab anyway, so let's merge this commit and I promise to fix the typo in another PR